providers/implementations/kdfs/hkdf.c

```diff
@@ -74,9 +75,18 @@ static int HKDF_Expand(const EVP_MD *evp_md,
     OSSL_PARAM_octet_string(OSSL_KDF_PARAM_KEY, NULL, 0),           \
     OSSL_PARAM_octet_string(OSSL_KDF_PARAM_SALT, NULL, 0)
 
-/* Gettable context parameters that are common across HKDF and the TLS KDF */
+/*
+ * Gettable context parameters that are common across HKDF and the TLS KDF
+ *   OSSL_KDF_PARAM_KEY is not gettable because it is a secret value
+ *   OSSL_KDF_PARAM_PROPERTIES is not gettable because, despite being listed as settable,
```

Comment:
This is incorrect.  `ossl_prov_digest_load_from_params` accesses `OSSL_KDF_PARAM_PROPERTIES`